cibuildwheel/options.py

```diff
@@ -362,13 +380,21 @@ def _inner_fmt(k: str, v: Any, table: TableFmt) -> Iterator[str]:
 
 
 class Options:
-    def __init__(self, platform: PlatformName, command_line_arguments: CommandLineArguments):
+    def __init__(
+        self,
+        platform: PlatformName,
+        command_line_arguments: CommandLineArguments,
+        environ: Mapping[str, str],
```

Comment:
Nitpick, but this is almost always called `env` rather than `environ` when passed and carried around (like in subprocess, for example), so I've usually called it `env` too. `environ` to me indicates the current environment, while `env` is a passed or modified environment. I am a fan of passing this around, but I like the `env` name a hair better, due to usage elsewhere.

Reply:
Ah, interesting. I don't usually use `environ` either, but I felt it was clearer when there are options called `environment` kicking around in the Options object. `environ` felt a lot closer to 'host environment' rather than 'environment option'. `env` is somewhere in the middle IMO.

Reply:
I've just had another look through the code and I can't see anywhere that the `environment` option and `env` would get confused. So I'll go with the more common convention.